### DISCUSSION TITLE
Overlay transcript on driver cam image

### DISCUSSION THREAD

**Discussion by jdschuitemaker** [f1multiviewer/issue-tracker#443] [Feature requests]
I would like to see an option to overlay the transcript of a driver on the camera image of that same driver. I have disabled transcripts, but for the driver I watch it would be a nice addition to see the communication on its image.

**Comment by xiphiasphotography** [comment]
Duplicate of #400 but this would be a great option. That keeps the transcript window clear for a few other drivers besides the OBC drivers.

**Comment by jeppeolesen** [comment]
Redirecting this to #400
